.github/workflows/ts.yml

```diff
@@ -0,0 +1,49 @@
+name: TS
```

Comment:
I added this file for TS ecosystem specific CI. In analogy to the "rust.yaml" workflow file.
I'm not sure yet, if it would make sense to merge them into one file at some point. This may open the doors for some optimization. Inside a workflow, hobs can depend on each other. Currently we run everything possible in parallel. Which is generally a good idea. But we may provoke earlier failure if we run fast-running and more likely to fail jobs first. We'll have to experiment with this.